tests/Integration/Integration.Tests/WorkstationModuleTests/WhenGettingAll.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Linn.Stores2.Integration.Tests.WorkstationModuleTests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+
+    using FluentAssertions;
+
+    using Linn.Stores2.Domain.LinnApps.Stores;
+    using Linn.Stores2.Integration.Tests.Extensions;
+    using Linn.Stores2.Resources.Stores;
+
+    using NUnit.Framework;
+
+    public class WhenGettingAll : ContextBase
```

Comment:
your module code suggests the get all route actually performs a search, so I would rename thie WhenSearching and pass query params and test things work that way